index.html

```diff
@@ -87,53 +90,47 @@
+            <h2>We pride ourselves on <strong>welcoming new members from all disciplines</strong> (you don't even need to know how to code, we promise). Our mission is to introduce how awesome technology is to the rest of the world, while creating cool things with it.<h2>
             </div>
+          <div class="col-lg-4" style="padding-bottom: 10px;"> <img src="img/hackercart.jpg" class="img-responsive img-rounded" ></div>            
         </div>
+        <h1>So why should you join us?</h1>
+<hr>
+        <div class="row">
+          <div class="col-lg-4"> <img src="img/hackathon.jpg" class="img-responsive img-rounded" ></div>
+          <div class="col-lg-8">
+            <h2>Well, first of all, we require like, <strong>0 commitment</strong> from you. You don't "join" hacsoc on account of we think that that whole club model is ridiculous. We know that you have classes and other commitments. All you have to do is show up when you want to, for the talks or events you're interested in.  </h2></div>
+ </div>
+ <div class="row">
+ <div class="col-lg-12">
+
+                        <h2><strong>If the beauty of a noncomittal relationship</strong> isn't enough for you, I'll bring up the fact that a ton of <strong>our club members and alumni work and intern at companies like Google, Microsoft, Amazon and various local startups</strong>. For recruiters, seeing "hacker society" on a resume means that that canditate goes above and beyond the classroom to get involved in the field. We're not the students that close our laptop after we get out of class- we're students who are excited and passionate about the applications of what we're learning.</h2>
+            <h2>The importance of getting involved in outside projects, hackathons, and other events is invaluable to your future career. Trust us. <strong>You'll learn stuff from chilling with us that you're not going to learn in class</strong> (your professors will generally be teaching theory, and we'll be showing you real-life application).</h2>
+
+            </div>     
+        </div>
+<h1>Gee whiz, sounds like a deal! Where do I sign, Mr. Berry Gordy?<h1>
+<hr>
+        <div class="row">
+          <div class="col-lg-4"> <img src="img/color.jpg" class="img-responsive img-rounded" ></div>
+          <div class="col-lg-8">
+          <h2>To get started, sign up for our <a href="https://lists.case.edu/wws/info/cwru-hackers?checked_cas=0&ticket=ST-5383-b7wjNUrejVJNcQ7gvB93-ssop-cfd-1.cwru.edu" style="font"><u>mailing list</u></a>. The way we operate is incredibly old school: <strong>we send out an newsletter/email thing at the beginning of the week</strong> with all the details of that week's events. The newsletter will also be posted here, but join the list anyway because that's where last-minute opportunities and job offers are sent out.</h2>
```

Comment:
This appears to be a CAS session token that you'll want to get rid of before deploying...